ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -136,8 +136,8 @@ tests:
       BASELINE_CAPABILITY_SET: None
       PERSISTENT_MONITORING: "false"
       TEST_FILTERS: ~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
-      UPGRADE_CHECK_RUN_TAGS: '@disconnected and not @console and not @storage'
```

Comment:
Shall we also remove the console tag on https://github.com/openshift/release/blob/e8eedfc0df5c362db263ad1ef58ddf65b9fd89fc/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml#L151 ?